keyboards/ymdk/ymd60/info.json

```
@@ -0,0 +1,228 @@
+{
+    "keyboard_name": "Oni HHKB",
+    "manufacturer": "YMDK",
+    "maintainer": "Pangorin",
+    "usb": {
+        "vid": "0x45D4",
+        "pid": "0x0021",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "ROW2COL",
+    "matrix_pins": {
+        "cols": ["F0", "F1", "E6", "C7", "C6", "B6", "D4", "B1", "F7", "B5", "B4", "D7", "D6", "B3", "B2"],
+        "rows": ["D0", "D1", "D2", "D3", "D5"]
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "features": {
+        "bootmagic": true,
+        "mousekey": false,
+        "extrakey": false,
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "rgb_matrix": {
+        "driver": "WS2812",
```

migrated from `config.h`
```suggestion
    },
    "ws2812": {
       "pin": "E2"
    },
    "rgb_matrix": {
        "driver": "WS2812",
        "max_brightness": 200,
```